packages/ra-input-rich-text/src/index.tsx

```diff
@@ -123,20 +157,13 @@ const RichTextInput = ({
     );
 };
 
-RichTextInput.propTypes = {
```

Comment:
need to keep the `propTypes`. Compile-time type validation doesn't replace runtime type checking (unfortunately). Would you mind adding these back?

Reply:
:+1: